docs/policy.md

```diff
@@ -202,3 +202,41 @@ policies:
 In the above example, the access to favicon is always granted and never requires an authorization.
 This feature is useful for web browsers and it is a good practice to set this policy.
 In the second policy, no-authorization access is granted to all member resources defined by a path wildcard.
+
+## Resource properties
+
+It is possible to filter allowed fields using `properties` or `hiddenProperties` properties.
+It is not possible to use both `properties` and `hiddenProperties` at the same time.
+- `properties` defines visible properties, the other properties are hidden.
+- `hiddenProperties` defines hidden properties, the other properties are visible.
```

Comment:
IMO, name `hiddenProperties` is confusing and works only with `read`. Consider changing it to `blacklistProperties` or maybe something else.

Reply:
I've change the name to `blacklistProperties`.